mentoring/choosing_a_solution.md

```diff
@@ -65,21 +74,21 @@ The screen looks like this:
 
 1. The main part on the left side contains the student's code.
    By default you'll be seeing their most recent iteration.
-   If they've submitted multiple iterations, view each iteration using the numbers in circles at the bottom. 
-
-   If they've only submitted one iteration, you won't see the number icons.
+   If they've submitted multiple iterations, you can switch through these using the numbers in circles at the bottom.
+   If they've only submitted one iteration, you won't see those icons.
 
 2. The tabs at the top shows the student's code, the tests, and the instructions.
    This is useful for reminding yourself about what the student has been asked to do in this exercise.
 
 3. There is an indicator showing whether a test has passed or failed.
    If they failed, click the indicator to open a report showing the specific details of their test and what they did wrong.
 
-On the right hand side, there are three tabs.
-- The overview tab will contain the same information about the user that you saw on the tooltip.
-- The student comment tab contains information from the user about what they want to learn from this specific solution.
-(For some older solutions, this might be missing).
-- The guidance tab provides the following information:
+On the right hand side, you'll see three tabs.
```

Comment:
Why are we removing the bullets?

Perhaps these should remain bullets (and not changed, as the writing that is here is without contractions and seem to be better without).  Also, I may or may not see the things described.  The description is good even without being told that I can see (which is sometimes not the case).